SharedSources/VLCMetadata.m

```diff
@@ -69,7 +68,7 @@ - (void)updateMetadataFromMediaPlayerForiOS:(VLCMediaPlayer *)mediaPlayer
     [self checkIsAudioOnly:mediaPlayer];
 
     if (self.isAudioOnly) {
-        self.artworkImage = [VLCThumbnailsCache thumbnailForManagedObject:item];
```

Comment:
This method was removed and we should add a ticket for adapting this part of the code to the new medialibrary